src/main/java/com/java_essential/Cherednichenko/extra_task/CustomerService/SupportService.java

```diff
@@ -0,0 +1,103 @@
+package com.java_essential.Cherednichenko.extra_task.CustomerService;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class SupportService {
+    Customer[] customers = new Customer[10];
+    Profile[] profiles = new Profile[13];
+    Service[] services = new Service[5];
+    SupportSpecialist[] supportSpecialists = new SupportSpecialist[3];
+    Tiket[] tikets = new Tiket[10];
+
+    public static void main(String[] args) {
+
+    }
+
+    public void fetch_all_customers() {
```

Comment:
```suggestion
    public void fetch_all_customers() {
```
method fetch_all_customers have incorrect naming, change it according to java code convention rules
see [https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html](url)